webpack/banner_size_warning_plugin.js

```diff
@@ -0,0 +1,60 @@
+const chalk = require( 'chalk' ).default;
+
+class BannerSizeWarningPlugin {
+	constructor( max_banner_size ) {
+		this.banner_size = max_banner_size.limit; // limit in bytes
```

Comment:
we unfortunately don't have linting on those config/tooling files,
but usually use camelCase for variable names, so you could change it to camelCase in here as well